src/codecs/music_gme.h

```diff
@@ -0,0 +1,35 @@
+// https://github.com/WohlSoft/SDL-Mixer-X
+
+/*
+  SDL_mixer:  An audio mixer library based on the SDL library
+  Copyright (C) 1997-2020 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+/* This file supports playing chiptune files with libGME */
+
+#include "music.h"
+
+extern Mix_MusicInterface Mix_MusicInterface_GME;
+
+#ifdef MUSIC_GME
+
+extern void _Mix_GME_SetSpcEchoDisabled(void *music, int disabled);
+extern int  _Mix_GME_GetSpcEchoDisabled(void *music);
```

Comment:
AFAICS, not referenced anywhere. What are these for?

Reply:
This is one of the most recent features I added to GME, but the original author seems away for any reason and he won't be able to review [my pull requests](https://bitbucket.org/mpyne/game-music-emu/pull-requests/) for a long time...

One note: at my CMake build there is a check for the presence of certain API functions at currently available GME assembly, and so, if using an old version, this function becomes a dummy.

Reply:
> specific to your fork 

yet, until original author will accept my pull-requests

Reply:
@sezero, two of these forks you mentioned, are very outdated (@kode54, author of one of them, can confirm). Also, you may see, that their code is totally different from the original in the case of repo organizing, way of building, internality, etc. Changes I did at my copy, are intended to be merged into mainstream, but they didn't because the current maintainer is not available. So, from here you may drop (or comment out until the thing gets merged into the mainstream once upon)